packages/arb-token-bridge-ui/src/components/MainContent/ArbitrumStats.tsx

```diff
@@ -11,15 +11,15 @@ import { useNetworkTPS } from '../../hooks/useNetworkTPS'
 export const statsLocalStorageKey = 'arbitrum:bridge:preferences:stats'
 
 const getActivityThresholdL1 = (gasPrice: number) => {
-  if (gasPrice < 20) return { className: 'text-[#008000]' }
-  if (gasPrice < 40) return { className: 'text-orange-arbitrum-nova' }
-  return { className: 'text-[#ff0000]' }
+  if (gasPrice < 20) return { className: 'text-lime-dark' }
+  if (gasPrice < 40) return { className: 'text-arb-nova-primary' }
+  return { className: 'text-error' }
 }
 
 const getActivityThresholdL2 = (gasPrice: number) => {
-  if (gasPrice < 0.5) return { className: 'text-[#008000]' }
-  if (gasPrice < 2) return { className: 'text-orange-arbitrum-nova' }
-  return { className: 'text-[#ff0000]' }
+  if (gasPrice < 0.5) return { className: 'text-lime-dark' }
+  if (gasPrice < 2) return { className: 'text-arb-nova-primary' }
+  return { className: 'text-error' }
```

Comment:
current:
![image](https://user-images.githubusercontent.com/13184582/235649698-895c5f97-def4-4101-b92c-c6d68630d24c.png)

previous:
![image](https://user-images.githubusercontent.com/13184582/235649758-994b02b9-b273-4952-80e6-92ed2a332bc0.png)

seems to be better to keep the previous colours

maybe we can work with Val to get a set of colours against dark background and a set against light background if we want to use green/yellow/red for dark bg

Reply:
IMO this should be decided by designer for the design system, then you can safely use it in the differnt repos